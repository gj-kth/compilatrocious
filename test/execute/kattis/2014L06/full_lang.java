// EXT:IWE
// EXT:BDJ
// EXT:CLE
// EXT:CGT
// EXT:CGE
// EXT:CEQ
// EXT:CNE

class Main2 {
    public static void main(String[] args) {
        Ext2 ext;

        if (10 > 2 && 5 == 5 && 7 != 2 && 10 >= 9 && 9 <= 10) {
            System.out.println(true);
        }

        if (5 == 6) {
            System.out.println(99);
        }

        if (5 != 5) {
            System.out.println(85);
        }

        if (9 >= 10) {
            System.out.println(73);
        }

        if (10 <= 9) {
            System.out.println(155);
        }

        if (false || true) {
            System.out.println(true || false);
        }

        ext = new Ext2();
        if (true || ext.printThis(7)) {
            System.out.println(8);
        }

        if (false || ext.printThis(9)) {
            System.out.println(10);
        }
    }
}

class Ext2 {
    public boolean printThis(int a) {
        System.out.println(a);
        return true;
    }
}