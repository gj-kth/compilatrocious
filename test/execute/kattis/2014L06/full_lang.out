true
true
8
9
10
